crypto/objects/obj_dat.c

```diff
@@ -239,72 +286,29 @@ ASN1_OBJECT *OBJ_nid2obj(int n)
     ad.type = ADDED_NID;
     ad.obj = &ob;
     ob.nid = n;
+    if (!ossl_obj_read_lock(lock))
+        return NULL;
```

Comment:
Raise an err? Same for the `added == NULL` check above

Reply:
Will add here and elsewhere.